CHANGELOG.md

```
@@ -1,677 +1,685 @@
-
-- Refactor RetryPolicyBuilder #2296
-- Improve log information during initialization process #2297
-- Add metrics about retries (related to RetryPolicyBuilder) #2298
-
-Install for demonstration:
-```
-docker pull keeps/roda:v4.5.1
-```
+# ETERNA Changelog
+## v0.6.1 (2026-04-20)
+#### Bug fixes
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)